### ISSUE TITLE
About 1.12 version

### ISSUE THREAD

**Issue by kappa-maintainer** [Gamepiaynmo/CustomPlayerModel#23]
Do you have plan to make 1.12 version open source? The current version still have many compatible problems, and I found patching it thorough asm is way harder than modifying the mod directly

**Comment by ghost** [comment]
https://github.com/Gamepiaynmo/CustomModel ?

**Comment by kappa-maintainer** [comment]
Yes, its repo doesn't contain a license, and is set to All Right Reserved on CurseForge

**Comment by Gamepiaynmo** [comment]
It is open-source, you can use it as you like.

**Comment by kappa-maintainer** [comment]
Thank you. You really should change its license on CurseForge, so more people can extend the mod's life.

**Comment by kappa-maintainer** [comment]
I am going to re-license my 1.12 fork under MIT like the remaked version, is that OK?
